### PR TITLE
lowercase MAC addresses for TFTP boot

### PR DESCRIPTION
**Pull request by TribalNightOwl**
Fixes Issue: PXE Unable to locate a configuration file caused by filename generator bug #132

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -236,15 +236,15 @@
...
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ bootstrap.macaddr | lower | regex_replace (':', '-')}}"
         mode: 0555
       notify:
         - restart tftp
 
     - name: Set the master specific tftp files
       template:
         src: ../templates/pxe-master.j2
-        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | regex_replace (':', '-')}}"
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | lower | regex_replace (':', '-')}}"
```
Comment (christianh814) [inline tasks/main.yml:247]: Doesn't the `lower` in the `with_items` section satisfy the requirement?

File: tasks/main.yml [hunk 2]
```diff
@@ -253,7 +253,7 @@
     - name: Set the worker specific tftp files
       template:
         src: ../templates/pxe-worker.j2
-        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | regex_replace (':', '-')}}"
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | lower | regex_replace (':', '-')}}"
```
Comment (christianh814) [inline tasks/main.yml:256]: Same as the masters section...

Doesn't the `lower` in the `with_items` section satisfy the requirement?

File: tasks/main.yml (hunk 1 above)
Comment (TribalNightOwl) [reply tasks/main.yml]: You are right. Fixed.

File: tasks/main.yml (hunk 2 above)
Comment (TribalNightOwl) [reply tasks/main.yml]: You are right. Fixed

**Pull request merged**